### DISCUSSION TITLE
Reducing extra steps in backlog grooming and ticket creation

### DISCUSSION THREAD

**Discussion by alexevanczuk** [community/community#41018] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Wanted to share two common ways I use GitHub projects frequently that I think could be made simpler. The points are in order of importance. Thank you for reading!

# Browser info
```
Safari, Version 16.1 (18614.2.9.1.12)
```

# Ability to create new issues with assigned repo
My team and I use GitHub projects to maintain our backlog. When creating tickets, I need to specify a repo each time. If it could just auto-populate a chosen repo on new ticket creation, that would be a huge help.

This issue exacerbates a separate issue, which is that if a colleague created an issue and didn't assign it to a repo, when I later come in to designate a repo, it sets me as the owner, with no clear indication that I wasn't the one who originally wrote the ticket. I usually need to come in and add a disclaimer at the top describing the original author. Am I doing something wrong? Ideally, it would keep the original owner and perhaps say "edited by: [ ... ]" with perhaps a modal that opens when clicked indicating what changed (similar to how I believe GitHub comment revision history works).

We don't even really have a strong need for assigning a ticket to a repo, but I'm sure it helps with being able to conveniently auto-link references to other issues/tickets/etc, so I understand why we need to – it would just be great if I could just choose a default for the board.

# Bulk move tickets
I'm unable to multi-select tickets to move them across the board. This is a bit of a bother because if we completed 50 tickets that week, I have to individually drag and drop 50 tickets to another column. A "select all" for a column, or even just a way I could multi-select, would save a lot of time.

# Ability to create new issues with assigned milestone
When working in the "table" (what I know as "swim lane") view, I can group tickets by milestone. I'd like to be able to create new tickets in a group and have them auto-assign to that grouping. Ideally, it can more generally create new tickets with the same properties of the other tickets, particularly if those properties constitute the grouping behavior for the swim lane.

# Thank you!

**Comment by ossanna16** [comment]
Hi there @alexevanczuk and welcome to our community! Thank you for the feedback!

**Comment by rileybroughten** [comment]
@alexevanczuk bulk edit support for dragging and dropping board items is now supported! Check out the [changelog](https://github.blog/changelog/2023-05-25-github-issues-projects-may-25th-update/#%F0%9F%92%AA-%E2%8C%A8%EF%B8%8F-bulk-updates-and-keyboard-navigation-on-boards) announcement ✨ We are also looking to add a menu option (`...` menu) to do with as well 😁
